R/run.R

```diff
@@ -22,33 +27,55 @@
 #'
 #' # run basic example within learnr
 #' \dontrun{run_tutorial("hello", "learnr")}
-run_tutorial <- function(name = NULL, package = NULL, shiny_args = NULL) {
+run_tutorial <- function(
+  name = NULL,
+  package = NULL,
+  shiny_args = NULL,
+  clean = FALSE
+) {
```

Comment:
Should we isolate the args after `...` to allow for easier parameter expansion in the far future? 

```suggestion
  ...,
  shiny_args = NULL,
  clean = FALSE
) {
  ellipsis::check_dots_unnamed()
```

I don't think many people are using `shiny_args` unnamed, so I believe it is an ok change.

Reply:
edit: updated to pick the right function: `check_dots_empty()`